internal/venice-client-common/src/main/java/com/linkedin/venice/compute/ComputeUtils.java

```diff
@@ -0,0 +1,344 @@
+package com.linkedin.venice.compute;
+
+import com.linkedin.avro.api.PrimitiveFloatList;
+import com.linkedin.venice.VeniceConstants;
+import com.linkedin.venice.compute.protocol.request.ComputeOperation;
+import com.linkedin.venice.compute.protocol.request.CosineSimilarity;
+import com.linkedin.venice.compute.protocol.request.Count;
+import com.linkedin.venice.compute.protocol.request.DotProduct;
+import com.linkedin.venice.compute.protocol.request.HadamardProduct;
+import com.linkedin.venice.compute.protocol.request.enums.ComputeOperationType;
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.utils.CollectionUtils;
+import com.linkedin.venice.utils.Pair;
+import com.linkedin.venice.utils.RedundantExceptionFilter;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * This class provides utilities for float-vector operations, and it also handles {@link PrimitiveFloatList}
+ * transparently to the user of this class.
+ */
+public class ComputeUtils {
```

Comment:
So this file is just renamed, no change, right?

Reply:
Yes, I just merged `com.linkedin.venice.utils.ComputeUtils` and `com.linkedin.venice.compute.ComputeOperationUtils` plus added `computeResult`, which is based on original implementation in `StorageReadRequestsHandler`.